cmd/ocm-backplane/console/console_test.go

```diff
@@ -1,189 +0,0 @@
-package console
```

Comment:
Why were the console tests removed entirely? If you intend to address/fix them in another PR, can you create a Jira and comment the Jira in this file commenting out the code?